desk/app/reel.hoon

```diff
@@ -229,21 +240,30 @@
     =/  has  (~(has in open-link-requests) [(slav %p ship.pole) name.pole])
     ``json+!>([%b has])
   ::
-      [%x any %metadata token=@ ~]
-    =/  =metadata:reel  (fall (~(get by our-metadata) token.pole) *metadata:reel)
+      [%x %v1 %metadata ship=@ name=@ ~]
+    =/  id  (rap 3 ship.pole '/' name.pole ~)
+    =/  token  (~(get by stable-id) id)
+    ?~  token  [~ ~]
+    =/  =metadata:reel  (fall (~(get by our-metadata) u.token) *metadata:reel)
+    ``reel-metadata+!>(metadata)
+  ::
+      [%x %v0 %metadata name=@ ~]
+    ::  old style tokens are directly in metadata
+    =/  id  (rap 3 (scot %p our.bowl) '/' name.pole ~)
+    =/  =metadata:reel  (fall (~(get by our-metadata) id) *metadata:reel)
     ``reel-metadata+!>(metadata)
   ::
       [%x any %token-url token=*]
-    =/  =token:reel  (crip (join '/' token.pole))
+    =/  =token:reel  (crip +:(spud token.pole))
```

Comment:
Is this meaningfully different from the `+join` implementation?

Reply:
this was giving me errors I think? don't quite remember